corgi/core/migrations/0107_copy_srpm_license_data_to_binary_rpms.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.20 on 2023-11-27 05:50
+
+from django.db import migrations
+from corgi.core.models import Component
+
+
+def copy_srpm_license_data_to_binary_rpms(apps, schema_editor) -> None:
+    for scanned_srpm in (
+        Component.objects.filter(type=Component.Type.RPM, arch="src")
+        .exclude(license_concluded_raw="")
+        .iterator()
+    ):
+        for binary_rpm in scanned_srpm.provides.filter(type=Component.Type.RPM).exclude(
+            license_concluded_raw=""
+        ):
```

Comment:
You might also want to use .iterator() here, and I'm not sure the logic is right. With the current .exclude() code, we'll skip binary RPMs which are missing a license, and only update binary RPMs which already had scan results reported separately by OpenLCS.

I think instead, we want to use .filter() to skip any binary RPMs where OpenLCS already uploaded a scan result (they do this for newer RPMs). Then we'd only copy the license from the SRPM to the binary RPM if OpenLCS didn't upload any scan result (older RPMs, before they made a change to their scanning process).
```suggestion
        for binary_rpm in scanned_srpm.provides.filter(
            type=Component.Type.RPM,
            license_concluded_raw="",
        ).iterator():
```